src/main/java/com/google/jenkins/plugins/computeengine/ComputeEngineInstance.java

```diff
@@ -23,37 +23,41 @@
 import hudson.slaves.AbstractCloudComputer;
 import hudson.slaves.AbstractCloudSlave;
 import hudson.slaves.ComputerLauncher;
-import hudson.slaves.NodeProperty;
 import hudson.slaves.RetentionStrategy;
 import java.io.IOException;
 import java.util.Collections;
-import java.util.Optional;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 import jenkins.model.Jenkins;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
 
+@Getter
+@Setter
 public class ComputeEngineInstance extends AbstractCloudSlave {
   private static final long serialVersionUID = 1;
   private static final Logger LOGGER = Logger.getLogger(ComputeEngineInstance.class.getName());
 
   // TODO: https://issues.jenkins-ci.org/browse/JENKINS-55518
-  public final String zone;
-  public final String cloudName;
-  public final String sshUser;
-  public final transient Optional<WindowsConfiguration> windowsConfig;
-  public final boolean createSnapshot;
-  public final boolean oneShot;
-  public Integer launchTimeout; // Seconds
+  private String zone;
```

Comment:
Why was the "final" modifier removed from these fields?

Reply:
Good catch, I think I did that for consistency with what was done elsewhere because Databound setters were required. For ComputeEngineInstance no setters are required and these can go back to being final.